AmplifyPlugins/API/Tests/APIHostApp/APIHostApp/SubscriptionView.swift

```diff
@@ -55,6 +55,7 @@ class SubscriptionViewModel: ObservableObject {
         self.todos.append(todo)
     }
 }
+@available(iOS 14.0, *)
```

Comment:
I'm assuming the answer is "no", but just to make sure. The `SubscriptionView` isn't necessary for any of the tests that are running on iOS 13?

Reply:
It looks like we don't need it based on the tests. If we do at some point, based on what `SubscriptionView` is doing, it looks like we can inject an `@ObservableObject` instead.

Reply:
I think it won't work with iOS13. But we are using [the latest OS version](https://github.com/aws-amplify/amplify-swift/blob/main/.github/composite_actions/run_xcodebuild_test/action.yml#L17) of [Github macos-12 Xcode](https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md), which should be iOS 16.2